pkg/containerutil/cp_linux.go

```diff
@@ -0,0 +1,179 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package containerutil
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/containerd/nerdctl/pkg/rootlessutil"
+	"github.com/containerd/nerdctl/pkg/tarutil"
+	securejoin "github.com/cyphar/filepath-securejoin"
+	"github.com/sirupsen/logrus"
+)
+
+// Cp implements `nerdctl cp`.
+//
+// See https://docs.docker.com/engine/reference/commandline/cp/ for the specification.
+func Cp(ctx context.Context, container2host bool, pid int, dst, src string, followSymlink bool) error {
```

Comment:
It's confusing to have 2 funcs with the same name. Maybe we should keep the original name `Kopy` (or just `Copy`)

Reply:
> Should be moved to `pkg/containerutil.Cp`

Sorry maybe I misunderstood this suggestion, I'll rename it back.